### DISCUSSION TITLE
How Long I have to wait to synchronaize my wallet?

### DISCUSSION THREAD

**Discussion by PanWladcaJakub** [Chia-Network/chia-blockchain#13904] [General]
My wallet is already synchronaizing 4 days, how long I have to wait yet? Will end ? ;D

**Comment by AndrewSavers** [comment]
The same questions

**Comment by wallentx** [comment]
Are you able to provide some info about your setup? What version are you using? What are the specs of the system you are using, and the OS? Are you using a local full node, or running in wallet mode? Do you know if you have a lot of "dust" transactions?

**Reply by PanWladcaJakub** [reply]
that version : ,,Version 1.6.2 | 64 bit
Alternative Download: [Torrent ](https://download.chia.net/latest/Setup-Win64.exe.torrent)
No Windows 7 support

SHA256 checksum:
1dae90f9022e088c066a3d398c0e726607ae57fdc7c10781455d3ae43848e3ec" 

- My sysytem is windows 11. 
- what is OS? 
- Are you using a local full node, or running in wallet mode? Do you know if you have a lot of "dust" transactions?
- dont know that. 
but I can see my tokens in my wallet and I hope I can send i it also? just It is not syncronised, do you know what I can do ?
